src/cpu/x64/jit_uni_reorder.cpp

```diff
@@ -179,7 +179,9 @@ struct jit_uni_reorder_kernel_f32_t : public kernel_t, public jit_generator {
                 && IMPLICATION(utils::one_of(bf16, p.itype, p.otype),
                         mayiuse(avx512_core) || mayiuse(avx2_vnni_2))
                 && IMPLICATION(utils::one_of(f16, p.itype, p.otype),
-                        mayiuse(avx512_core_fp16) || mayiuse(avx2_vnni_2))
+                /* WA: We relax the isa condition here to restore (possibly to improve) the model compilation time after moving the f16 constant folding
```

Comment:
Comments inside the code are not required. Commit message is enough

Reply:
done, thanks